src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/ast.go

```diff
@@ -591,6 +603,7 @@ const (
 	GlobalTableType         = "globaltable"
 	PartitionTableHash      = "partitiontableHash"
 	NormalTableType         = "normaltable"
+	PartitionTableList      = "partitionTableList"
```

Comment:
The enum here is strange, I prefer to lowcase as:
globaltable
partitiontablehash
normaltable
partitiontablelist

Reply:
ok